compiler/rustc_ast/src/token.rs

```diff
@@ -454,6 +454,7 @@ impl Token {
             || self.is_keyword(kw::Union)
             || self.is_keyword(kw::Macro)
             || self.is_keyword(kw::MacroRules)
+            || self == &OpenDelim(Delimiter::Parenthesis)
```

Comment:
What..exactly is this for?

Reply:
I was reading, found out there is a this kind of scenario:

```rust
 // This function is visible to the entire crate
        pub(crate) fn crate_visible_fn() {}

        // This function is visible within `outer_mod`
        pub(super) fn super_mod_visible_fn() {
            // This function is visible since we're in the same `mod`
            inner_mod_visible_fn();
        }

```

```rust
OpenDelim(Delimiter::Parenthesis)
```
is used for expecting an '(', I tested it.

https://doc.rust-lang.org/reference/visibility-and-privacy.html?highlight=pub#visibility-and-privacy

Reply:
do you think we need to support this case? 
I just don't know `pub` could be used for so many kind of items and scenarios...😊

Reply:
Ohh, this is fine, then.